Build failures in translate/automl

It seems like a deterministic failure.

[Python 3.6 build](https://source.cloud.google.com/results/invocations/0bdc07d0-caac-4b1a-861e-108b8b364441/targets)
[Python 3.7 build](https://source.cloud.google.com/results/invocations/6b6b1774-e53c-4f02-843a-66918d090f03/targets)
[Python 3.8 build](https://source.cloud.google.com/results/invocations/e3b915f1-0aab-4e64-bd71-970beebcf7fb/targets)

Log:
```
_________________________ test_model_list_get_evaluate _________________________
Traceback (most recent call last):
  File "/workspace/translate/automl/model_test.py", line 80, in test_model_list_get_evaluate
    assert "evaluation_metric" in out
AssertionError: assert 'evaluation_metric' in 'name: "projects/1012616486416/locations/us-central1/models/VAR6465236123462402048/modelEvaluations/452876416420741254"\ncreate_time {\n  seconds: 1595365133\n  nanos: 496593000\n}\nevaluated_example_count: 30\n\n'
```

BTW, Build Cop Bot didn't report them. I filed:
https://github.com/googleapis/repo-automation-bots/issues/727

--- a/translate/automl/dataset_test.py
+++ b/translate/automl/dataset_test.py
@@ -14,8 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import datetime
 import os
+import uuid
 
 import pytest
 
@@ -24,11 +24,10 @@ import automl_translation_dataset
 project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
 compute_region = "us-central1"
 
-
 @pytest.mark.slow
 def test_dataset_create_import_delete(capsys):
     # create dataset
-    dataset_name = "test_" + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+    dataset_name = f"test_{uuid.uuid4().hex}"
     automl_translation_dataset.create_dataset(
         project_id, compute_region, dataset_name, "en", "ja"
     )

--- a/translate/automl/dataset_test.py
+++ b/translate/automl/dataset_test.py
@@ -24,10 +24,11 @@ import automl_translation_dataset
 project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
 compute_region = "us-central1"
 
+
 @pytest.mark.slow
 def test_dataset_create_import_delete(capsys):
     # create dataset
-    dataset_name = f"test_{uuid.uuid4().hex}"
+    dataset_name = f"test_{uuid.uuid4().hex[:27]}"
     automl_translation_dataset.create_dataset(
         project_id, compute_region, dataset_name, "en", "ja"
     )

--- a/translate/automl/model_test.py
+++ b/translate/automl/model_test.py
@@ -77,4 +77,4 @@ def test_model_list_get_evaluate(capsys):
         project_id, compute_region, model_id, model_evaluation_id
     )
     out, _ = capsys.readouterr()
-    assert "evaluation_metric" in out
+    assert model_evaluation_id in out

